web/src/js/actions.js

```diff
@@ -117,6 +117,13 @@ export var FlowActions = {
     },
     clear: function(){
         $.post("/clear");
+    },
+    download: () => window.location = "/flows/dump",
+    upload: (file) => {
+        var filereader = new FileReader();
+        filereader.file = file;
+        filereader.onload = (e) => {$.post("/flows/dump", e.target.result); e.preventDefault();};
+        filereader.readAsBinaryString(file);
     }
```

Comment:
This looks good. At some point we probably want to kick jQuery out and use the HTML5 fetch API directly.
Do you want to include this right away?

https://stackoverflow.com/questions/36067767/how-do-i-upload-a-file-with-the-html5-js-fetch-api